views/index.ejs

```diff
@@ -8,9 +8,9 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= products.length %></strong>
```

Comment:
The name of the array holding all the products is cart. So you'll want the length of the cart here, not products.